recipe/meta.yaml

```diff
@@ -64,7 +64,7 @@ test:
 
     - Sherpa --help
     - cd Examples/CI/LO_Z
-    - Sherpa
+    - Sherpa  # [build_platform == target_platform]
```

Comment:
Still want to use

```
test: native_and_emulated
```

in `conda-forge.yml` so the parts above that aren't runtime evaluations get tested.